app/config/config.yml

```diff
@@ -202,6 +202,7 @@ doctrine:
 swiftmailer:
     transport: %mailer_transport%
     host:      %mailer_host%
+    port:      %mailer_port%
```

Comment:
Why remove that? It broke the build...

https://travis-ci.org/redelivre/login-cidadao/builds/416593768#L744